tests/integration/test_emergency_contacts.py

```diff
@@ -113,19 +104,17 @@ def test_emergency_contacts_PUT(client, auth_headers):
     # TODO: Create issue to address non restful behavior.
 
     id = 100
-    response = client.put(
-        f"{endpoint}/{id}", json=updatedInfo, headers=auth_headers["admin"]
-    )
+    response = client.put(f"{endpoint}/{id}", json=updatedInfo, headers=valid_header)
     assert is_valid(response, 404)  # NOT FOUND
     assert response.json == {"message": "Emergency contact not found"}
 
 
-def test_emergency_contacts_DELETE(client, auth_headers):
+def test_emergency_contacts_DELETE(test_database, client, valid_header):
```

Comment:
I don't think we need to pass test_database here because it is already running given the nature of how these tests are setup.

Reply:
haha nevermind, I forget it was because of auth_headers is why test_database was not needed before.